walker/walk_test.go

```diff
@@ -1,3 +1,5 @@
+// +build aix darwin dragonfly freebsd js,wasm linux netbsd openbsd solaris
```

Comment:
The following tests look the same as Windows. So, we don't need this build tag, right? In other words, do these tests pass even on Windows?